### PR TITLE
PLAT-123245: Fixed ExpandableList focus moves to unexpected position on closing when selecting an item via touch input

### PR DESCRIPTION
**Pull request by SkylerBaek**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed

* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)

The focus moves to an unexpected position on closing ExpandableList when selecting an item via touch input

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

Since 'focus' event occurs after 'onTap' callback, focusing occurs during ExpandableList is closing.
To resolve this case, I made ExpandableList to be closed when 'onClick' occurs.

### Links
[//]: # (Related issues, references)
PLAT-123245

Enact-DCO-1.0-Signed-off-by: Seungcheon Baek (sc.baek@lge.com)

### CODE REVIEW

**Review by ybsung — APPROVED** [review]
LGTM

**Comment by MikyungKim** [comment]
@0x64 Would you please add qa-sampler for this case?

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ The following is a curated list of changes in the Enact moonstone module, newest
 
 ### Fixed
 
+- `moonstone/ExpandableList` to keep focus position properly when selecting item by touch
```
Comment (MikyungKim) [inline CHANGELOG.md]:
```suggestion
- `moonstone/ExpandableList` to keep focus properly when selecting an item by touch
```

**Review by MikyungKim — APPROVED** [review]
:lgtm:

**Pull request merged**